index.html

```diff
@@ -174,7 +174,57 @@ <h4 class="about">About me</h4>
         </ul>
     </div>
         </section>
+
+        <section class="form">
+            <h2 class="prg">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h2>
+          
+                <form  action="https://formspree.io/f/xpzbqzab" method="post">
+                    <input
+                    class="input"
+                    type="text"
+                    name="name"
+                    id="name"
+                    placeholder="Full name"
+                    maxlength="30"
+                    required/>
+                
+                <hr>
+                
+                  <input
+                    class="input"
+                    type="email"
+                    name="email"
+                    id="email"
+                    placeholder="Email address"
+                    required
+                  />
+                
+                <hr>
+                
+                  <textarea class="msg"
+                    name="message"
+                    id="message"
+                    cols="30"
+                    rows="10"
+                    placeholder="Enter text here"
+                    maxlength="500"  
+                    required   
+                  ></textarea>
+                </form>
+                
+                <div class="tonic">
+                    <button class="btn-tonic" type="submit">Get In Touch</button>
+                </div>   
```

Comment:
- [ ] Just as the previous review suggested, your form is not been submitted successfully. I think it's because you place the submit button out of the `form` element. Try placing the enter submit button element inside the `form` element just after the `textarea` and try it out.
Please verify that the form submits without errors before asking for a re-review.